### PR TITLE
Feature/sg 189 take yolov5 and yolov3 out

### PR DESCRIPTION
**Pull request by shaydeci**

### CODE REVIEW

File: src/super_gradients/examples/coco_eval_script/COCO_eval_script.py [hunk 1]
```diff
@@ -0,0 +1,748 @@
+import json
```
Comment (ofrimasad) [inline src/super_gradients/examples/coco_eval_script/COCO_eval_script.py:1]: I think this script went into the PR by mistake...

File: src/super_gradients/examples/yolov5_transfer_learning_example/yolov5_transfer_learning_example.py [hunk 2]
```diff
@@ -1,102 +0,0 @@
-"""
```
Comment (ofrimasad) [inline src/super_gradients/examples/yolov5_transfer_learning_example/yolov5_transfer_learning_example.py:1]: don't you prefer fixing this with Yolox?

File: src/super_gradients/recipes/arch_params/yolo_arch_params.yaml [hunk 3]
```diff
@@ -23,7 +23,7 @@ add_nms: False  # Add the NMS module to the computational graph
 nms_conf: 0.25  # When add_nms is True during NMS predictions with confidence lower than this will be discarded
 nms_iou: 0.45  # When add_nms is True IoU threshold for NMS algorithm
                # (with smaller value more boxed will be considered "the same" and removed)
-yolo_type: 'yoloV5'   # Type of yolo to build: 'yoloV5' and 'yoloX' are supported
+yolo_type: 'yolox'   # Type of yolo to build: 'yoloV5' and 'yoloX' are supported
```
Comment (ofrimasad) [inline src/super_gradients/recipes/arch_params/yolo_arch_params.yaml:26]: fix comment as well

**Review by ofrimasad — COMMENTED** [review]
looks good. just some minor comments

File: src/super_gradients/examples/coco_eval_script/COCO_eval_script.py (hunk 1 above)
Comment (shaydeci) [reply src/super_gradients/examples/coco_eval_script/COCO_eval_script.py]: removed.

File: src/super_gradients/examples/yolov5_transfer_learning_example/yolov5_transfer_learning_example.py (hunk 2 above)
Comment (shaydeci) [reply src/super_gradients/examples/yolov5_transfer_learning_example/yolov5_transfer_learning_example.py:1]: its a bad idea to keep them inside the repo IMO.

File: src/super_gradients/recipes/arch_params/yolo_arch_params.yaml (hunk 3 above)
Comment (shaydeci) [reply src/super_gradients/recipes/arch_params/yolo_arch_params.yaml]: fixed.

**Review by ofrimasad — APPROVED** [review]
LGTM

**Pull request merged**